dev-docs/publisher-api-reference.md

```diff
@@ -445,6 +446,35 @@ Use this method to get all of the bids that have won their respective auctions b
 
 <hr class="full-rule">
 
+<a name="module_pbjs.getTargeting"></a>
+
+### pbjs.adserver.freewheel.getTargeting(options) ⇒ Object
+
+Use this method to get targeting key-value pairs to be sent to the ad server. 
+
++ `pbjs.adserver.freewheel.getTargeting(options)`: returns key-value pair from the ad server. 
+
+```Javascript
+
+pbjs.adservers.freewheel.getTargeting({
+    codes: [adUnitCode1],
+    callback: function(err, targeting) { 
+        //pass targeting to player api 
+    }
+});
+```
+#### Argument Reference
+
+The options object
+
+{: .table .table-bordered .table-striped }
+| Param | Scope | Type | Description |
+| --- | --- | --- | --- |
+| code | Optional | `Array` |  [`adUnitCode1`] |
```

Comment:
s/b codes, plural, per the code sample above. Also, Description seems to be move of an example than an actual description.

Reply:
Fixed.